meinberlin/apps/kiezradar/models.py

```diff
@@ -156,3 +166,42 @@ class Meta:
 
     def __str__(self):
         return f"kiezradar search profile - {self.name}, disabled {self.disabled}"
+
+
+def get_search_profiles_for_project(project: Project) -> QuerySet[SearchProfile]:
+    status = 2
+    if project.phases.active_phases():
+        status = 0
+    if project.phases.future_phases():
```

Comment:
I haven't looked at the methods but wouldn't this mean that if a project is active and also has a future phase it would be considered "future" and not "ongoing/active". Would `elif` make more sense?

Reply:
ahh, good catch, it was a return previously.